frontend/src/common/node.ts

```diff
@@ -50,3 +50,29 @@ export const mapSustainabilityGoalToNode = (sdg: SustainabilityGoal): Node | nul
   node.name = sdg.label;
   return node;
 };
+
+export const mapCorrelationToName = (correlation: number) => {
+  switch (correlation) {
+    case 2:
+      return 'høy';
+    case 1:
+      return 'medium';
+    case 0:
+      return 'lav';
+    default:
+      return '';
+  }
+};
+
+export const correlationToColor = (correlation: number) => {
```

Comment:
```suggestion
export const mapCorrelationToColor = (correlation: number) => {
```
To match the naming convention of the function above